substrate/frame/child-bounties/src/lib.rs

```diff
@@ -906,12 +929,15 @@ impl<T: Config> Pallet<T> {
 // where we pass the active child bounties and child curator fees to the parent
 // bounty.
 impl<T: Config> pallet_bounties::ChildBountyManager<BalanceOf<T>> for Pallet<T> {
+	/// Returns number of active child bounties for `bounty_id`
 	fn child_bounties_count(
 		bounty_id: pallet_bounties::BountyIndex,
 	) -> pallet_bounties::BountyIndex {
 		ParentChildBounties::<T>::get(bounty_id)
 	}
 
+	/// Returns cumulative child bounty curator fees for `bounty_id` also removing the associated
+	/// storage item. This function is assumed to be called when parent bounty is claimed.
```

Comment:
I was talking about the doc for the contract/trait not for the impl

Reply:
I wasn't sure it makes sense to document the trait itself instead of the functions where the action happens

Reply:
if you describe it in more general way it makes more sense. you set the contract, describing that it should behave like `take`, not like `get`.